api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_stats.py

```diff
@@ -19,6 +19,7 @@ def mean_variance(
     calculate_mean: bool = False,
     calculate_variance: bool = False,
     ddof: int = 1,
+    exclude_zeros: bool = False,
 ) -> pd.DataFrame:
     """
     Calculate  mean and/or variance along the ``obs`` axis from query results. Calculations
```

Comment:
the docstrings lack documentation of the `exclude_zeros` argument

Reply:
Oops, forgot to add that. Thanks!